v2/src/log_context_property_basic_types.c

```diff
@@ -50,8 +92,22 @@
         else \
         { \
             /* Codes_SRS_LOG_CONTEXT_PROPERTY_BASIC_TYPES_01_007: [ LOG_CONTEXT_PROPERTY_TYPE_IMPL(int64_t).copy shall copy the bytes of the int64_t value from the address pointed by src_value to dst_value. ]*/ \
+            /* Codes_SRS_LOG_CONTEXT_PROPERTY_BASIC_TYPES_01_017: [ LOG_CONTEXT_PROPERTY_TYPE_IMPL(uint64_t).copy shall copy the bytes of the uint64_t value from the address pointed by src_value to dst_value. ]*/ \
+            /* Codes_SRS_LOG_CONTEXT_PROPERTY_BASIC_TYPES_01_027: [ LOG_CONTEXT_PROPERTY_TYPE_IMPL(int32_t).copy shall copy the bytes of the int64_t value from the address pointed by src_value to dst_value. ]*/ \
+            /* Codes_SRS_LOG_CONTEXT_PROPERTY_BASIC_TYPES_01_037: [ LOG_CONTEXT_PROPERTY_TYPE_IMPL(uint32_t).copy shall copy the bytes of the uint64_t value from the address pointed by src_value to dst_value. ]*/ \
+            /* Codes_SRS_LOG_CONTEXT_PROPERTY_BASIC_TYPES_01_047: [ LOG_CONTEXT_PROPERTY_TYPE_IMPL(int16_t).copy shall copy the bytes of the int64_t value from the address pointed by src_value to dst_value. ]*/ \
+            /* Codes_SRS_LOG_CONTEXT_PROPERTY_BASIC_TYPES_01_057: [ LOG_CONTEXT_PROPERTY_TYPE_IMPL(uint16_t).copy shall copy the bytes of the uint64_t value from the address pointed by src_value to dst_value. ]*/ \
+            /* Codes_SRS_LOG_CONTEXT_PROPERTY_BASIC_TYPES_01_067: [ LOG_CONTEXT_PROPERTY_TYPE_IMPL(int8_t).copy shall copy the bytes of the int64_t value from the address pointed by src_value to dst_value. ]*/ \
+            /* Codes_SRS_LOG_CONTEXT_PROPERTY_BASIC_TYPES_01_077: [ LOG_CONTEXT_PROPERTY_TYPE_IMPL(uint8_t).copy shall copy the bytes of the uint64_t value from the address pointed by src_value to dst_value. ]*/ \
             *(type_name*)dst_value = *(type_name*)src_value; \
```

Comment:
>*(type_name*)dst_value = *(type_name*)src_value; [](http://example.com/codeflow?start=12&length=48)

So this is the "shall copy the bytes". Consider renaming to "shall assign" :)

Reply:
Oh, yes, I forgot I did the assign. I'll actually change this to memcpy.